2.2/.sti/bin/assemble

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+function rake_assets_precompile() {
+  [ -n "$DISABLE_ASSET_COMPILATION" ] && return
+  [ ! -f Gemfile ] && return
+  [ ! -f Rakefile ] && return
+  ! grep " rails " Gemfile.lock >/dev/null && return
+  ! grep " execjs " Gemfile.lock >/dev/null && return
+  ! bundle exec 'rake -T' | grep "assets:precompile" >/dev/null && return
+
+  echo "---> Starting asset compilation."
```

Comment:
I'll prepare new PR fixing this issue once this is merged. Lets keep it consistent and the commits atomic.